### DISCUSSION TITLE
Is there a way to set a fixed timerange for brightness while keeping color temperature circadian?

### DISCUSSION THREAD

**Discussion by ccbell42** [basnijholt/adaptive-lighting#840] [Q&A]
Is there a way to have this use a fixed schedule for brightness, regardless of the time of year, while still setting the color temperature based on the sun?

The sun is setting at 5pm right now. I like having my lights change to a warmer color when the sun goes down, but they're also get too dim too early. Ideally, I would have the lights at 100% brightness between 8am and 8pm, gradually dim to minimum brightness from 8pm to 11pm, stay at minimum until 5am, then gradually increase brightness back to max at 8am, year-round.

**Comment by frasderp** [comment]
I think this function is managed using brightness_mode with "linear" and "tanh". Have a read through the readme on the front page, there are a lot of examples of this.
